prod/westeurope/eucovidcert/functions_eucovidcert/function_app_slot_staging/terragrunt.hcl

```diff
@@ -0,0 +1,89 @@
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+# Common
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../common/key_vault"
+}
+
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v3.0.3"
```

Comment:
use app service slot module, see others function

Reply:
Done in [33ca10c](https://github.com/pagopa/io-infrastructure-live-new/pull/529/commits/33ca10ce7b6c70fb338f34d180f670d1c3d7a30f)